PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+lorem ipsum lorem ipsum lorem ipsum lorem
```

Comment:
Is this supposed to be filled here?

Reply:
@mariomerco This one, we leave it as is because it is intended to provide guidelines to anyone who would wish to contribute and open a pull request for this module.